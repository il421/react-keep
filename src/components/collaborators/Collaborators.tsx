import React from "react";
import CollaboratorForm from "./CollaboratorForm";
import { CollaboratorsList } from "./CollaboratorsList";

interface CollaboratorsProps {}

export class Collaborators extends React.PureComponent<CollaboratorsProps> {
  render() {
    return (
      <>
        <CollaboratorForm />
        <CollaboratorsList />
      </>
    );
  }
}
