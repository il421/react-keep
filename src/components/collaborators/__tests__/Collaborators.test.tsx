import { shallow, ShallowWrapper } from "enzyme";
import toJson from "enzyme-to-json";
import React from "react";

import { Collaborators } from "../Collaborators";

let wrapper: ShallowWrapper;

test("should render Collaborators", () => {
  wrapper = shallow(<Collaborators />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
