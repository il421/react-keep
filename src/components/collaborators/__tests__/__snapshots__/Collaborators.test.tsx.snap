// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Collaborators 1`] = `
<Fragment>
  <Connect(CollaboratorForm) />
  <Connect(CollaboratorsListBase) />
</Fragment>
`;
